package.json

```diff
@@ -13,9 +13,11 @@
     "dependencies": {
         "express": "^4.17.1",
         "hbs": "^4.1.2",
-        "request": "^2.88.2"
+        "prettier": "^2.5.1",
+        "request-promise-native": "^1.0.9"
     },
     "devDependencies": {
+        "dotenv": "^10.0.0",
```

Comment:
dotenv should be in `dependencies` block, prettier  - `devDependencies`

Reply:
Fixed.